tapcfg/config.go

```diff
@@ -254,7 +258,7 @@ type Config struct {
 	BatchMintingInterval time.Duration `long:"batch-minting-interval" description:"A duration (1m, 2h, etc) that governs how frequently pending assets are gather into a batch to be minted."`
 
 	// The following options are used to configure the proof courier.
-	ProofCourierMode string                    `long:"proofcouriermode" choice:"hashmail" description:"Type of proof courier to use."`
+	ProofCourierAddr *proof.CourierAddr        `group:"proofcourier" long:"proofcourier" description:"Proof courier service address."`
```

Comment:
I think this needs a `group:"proofcourier"` tag. And to make it look similar to the other grouped configs we have, maybe remove the `long:""` and `description:""` tags?

Reply:
> I think this [doesn't] need a...

I think you're saying I should remove `group:"proofcourier"`?

And the same for `HashMailCourier` just below it?

Reply:
Sorry, I meant it needs a `namespace:""` tag, same as the hash mail courier below (or `Lnd`).